boilerplate/app/screens/DemoDebugScreen.tsx

```diff
@@ -22,17 +22,19 @@ export const DemoDebugScreen: FC<DemoTabScreenProps<"DemoDebug">> = function Dem
 
   const demoReactotron = React.useMemo(
     () => async () => {
-      console.tron.display({
-        name: "DISPLAY",
-        value: {
-          appId: Application.applicationId,
-          appName: Application.applicationName,
-          appVersion: Application.nativeApplicationVersion,
-          appBuildVersion: Application.nativeBuildVersion,
-          hermesEnabled: usingHermes,
-        },
-        important: true,
-      })
+      if (__DEV__) {
```

Comment:
Hm is this a regression in ease of use, though? Each time you'd want to console.tron.x we'd have to wrap it in this check?

Reply:
Yeah, but according to the docs, Reactotron should really be disabled in production. So we need some mechanism to do that: https://github.com/infinitered/reactotron/blob/master/docs/tips.md#react-native

Ideally, it would be nice to have some developer tooling around this. There a babel plugin to strip Reactotron out of production builds: https://github.com/infinitered/babel-plugin-ignite-ignore-reactotron

We could also write an eslint rule

Reply:
is there a way to wrap it under the hood in `__DEV__`? the babel plugin is helpful but i had never heard of it

Reply:
The alternative is that we can stub out `console.tron` calls like we do currently. We currently add all the keys manually, but I think a proxy like this would be more resilient:

```
// Make sure the console.tron object exists
console.tron = console.tron || {};

// Create a handler for the proxy
let handler = {
    get: function(target, prop, receiver) {
        // If the property accessed is a function, return a noop function
        if (typeof target[prop] === 'function') {
            return function() {}; // noop function
        }
        // If the property is not a function, just return it
        return Reflect.get(...arguments);
    }
};

// Create a Proxy around console.tron object
console.tron = new Proxy(console.tron, handler);
``` 

We'd need to add this into the `else` block in our App entry point instead of creating the Reactotron config.

Reply:
This is also something we could ship in `reactotron` so that way we could be like
```js
if (__DEV__) {
  require('ReactotronConfig')
} else {
  require('reactotron-react-native/console-tron-stub') // add a source file to the package with the above stub
}

```

Reply:
The proxy is pretty cool and getting it for free from reactotron seems like a cool win as well (and can always come later)